spice/unattended/RHEL-7-spice.ks

```diff
@@ -50,7 +50,7 @@ firstboot --disable
 #
 # Packages.
 #
-%packages --default
+%packages --default --ignoremissing
```

Comment:
This will lead to uncontrolled list growing.
Previous KS had entries that was useless.
Okay, anyway, thanks for contributing.